tools/topology/CMakeLists.txt

```diff
@@ -25,6 +25,25 @@ ${stdout}")
 endfunction()
 
 
+# Being written in C, `alsatplg` silently ignores some invalid inputs
+# and produces an corrupt .tplg file instead of returning an error code
+# that fails the build.  For instance, alsatplg versions < 1.2.5 silently
+# corrupt `codec_consumer` and turn it into `codec_master` instead.
```

Comment:
nit: some grammer tense issues

Reply:
Thanks for the prompt review!

Could you be more precise about the grammar issues?

I found "produces A corrupt..." and nothing else (and that's not a tense). Google docs and MS Word don't find anything either.